components/automate-deployment/pkg/server/upgrade_status.go

```diff
@@ -24,35 +26,68 @@ import (
 // I think this should get a lot more simple as we integrate the
 // converger. In that world we can potentially have a function that
 // takes an expected state and tells us if it is true or not.
```

Comment:
We may remove this comment if not needed.

Reply:
this comment is needed for reference.